### PR TITLE
Config SMTP to Enable User Registration

### LINKED ISSUE #834: Enable user registration function by adding SMTP service configuration
- [ ] Use Terraform script to add SMTP service configuration in Zitadel service
- [ ] Add documentation explaining how to configure SMTP service for user registration.

### PR DESCRIPTION
**Pull request by baixinsui**
fixed https://github.com/eclipse-xpanse/xpanse/issues/834

### CODE REVIEW

File: zitadel/local/README.md [hunk 1]
```diff
@@ -47,39 +47,42 @@ instance, and download the credential file for the authentication which needed f
 
 The default IAM admin user login logged in successfully, click on `Users`,`Service User`, and `+ New N` items on the console
 in sequence to open page `Create a New User`.
-![open-create-service-user.png](compose/statics/open-create-service-user.png)
+![create_service_user_1](statics/create_service_user_1.png)
```
Comment (swaroopar) [inline zitadel/local/README.md]: pls rename statics folder to just `static`

File: zitadel/local/README.md [hunk 2]
```diff
@@ -89,40 +92,73 @@ scripts. What we need to pay attention to completing as below:
...
+![view_terraform_outputs](statics/view_terraform_outputs.png)
 
 The default IAM admin user login the console of Zitadel to view added configurations:
-![view-configs-on-console](compose/statics/view-configs-on-console.png)
+![view_org_project_apps](statics/view_org_project_apps.png)
 
-### Add Admin User of Xpanse
+## Add Users for Xpanse
+
+### Manual Configuration on Console
```
Comment (swaroopar) [inline zitadel/local/README.md]: this topic does not belong to `Add Users for Xpanse` topic. 
Also this topic is needed due to a bug in the provider. We can document that too. 
https://github.com/zitadel/terraform-provider-zitadel/issues/75

File: zitadel/local/README.md [hunk 3]
```diff
@@ -179,8 +216,8 @@ npm start
 ```
 
 After the service is successfully started, open internet browser and navigate to http://localhost:3000. The browser will
-redirect to the login page of the local service of Zitadel. Login with the created xpanse admin user.
-![login-with-xpanse-admin](compose/statics/login-with-xpanse-admin.png)
+redirect to the login page of the local service of Zitadel. Login with the created admin user of xpanse.
+![xpanse_admin_suer_login](statics/xpanse_admin_suer_login.png)
```
Comment (swaroopar) [inline zitadel/local/README.md]: typo in file name.. `suer` -> `user`

File: zitadel/local/README.md [hunk 4]
```diff
@@ -36,9 +36,9 @@ admin users login:
 
 > Note: The first login requires a password change, please remember the new password.
 
-## Run Terraform Scripts to initialize the configuration for Applications.
+## Initialize the configuration for Applications.
```
Comment (swaroopar) [inline zitadel/local/README.md]: I see that many configurations, including the SMTP is valid not only for local but also for other enviornments. 
Infact, we need SMTP for non-local only. For local environments, we must show how to directly add users. 

Its better we copy the documentation or have one common documentation and have references from there to the local readme.md and to a new readme file directly under zitadel folder. 

Please note there is one more readme under terraform folder. 

If you wish, we can also merge all of them into one single readme folder directly under zitadel folder.

**Review by swaroopar — CHANGES_REQUESTED** [review]

File: zitadel/local/README.md (hunk 4 above)
Comment (baixinsui) [reply zitadel/local/README.md]: Ok. I will merge all of them into the readme.md under the zitadel folder.

**Review by swaroopar — APPROVED** [review]
LGTM

**Pull request merged**